libs/template/themes/src/fes.theme.ts

```diff
@@ -8,6 +8,13 @@ export const fesTheme: Theme = {
   designTokens: () =>
     import('../design-tokens/src/backoffice').then((s) => s.backofficeTokens),
   icons: {
-    resource: materialDesignIcons,
+    resource: fontawesomeIcons,
+    resources: [
+      {
+        resource: backofficeIcons,
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        types: Object.keys(backofficeIcons.mapping!),
+      },
+    ],
```

Comment:
Can we keep the "fes" theme inside the fes repo? It's not a secret, but will make it easier to change things going forward. For example, we use currently fontawesome, but we do like to move to material icons

Reply:
I'd also deprecate the fes theme here, so we can drop it with upcoming major.

Reply:
fes theme updates inside fes repo: https://github.com/spryker/fes/pull/1969